src/components/DocumentListItem/DocumentListItem.js

```diff
@@ -1,8 +1,7 @@
 import React, { useState } from "react";
-import "font-awesome/css/font-awesome.min.css";
 import "./DocumentListItem.css";
-import PropTypes from "prop-types";
-import { formatDate } from "./../../util/dateFormatter";
+import { formatDate } from "../../util/dates";
+import Icon from "../Icon";
```

Comment:
I created an Icon component to centralize Icon handling for consistency and reuse.

Reply:
In progress